### PR TITLE
adding user archive param

### PR DESCRIPTION
**Pull request by manas-vessel**

### CODE REVIEW

File: docs/pages/crm/crm.yaml [hunk 1]
```diff
@@ -5426,6 +5432,9 @@ components:
         modifiedTime:
           type: string
           description: The timestamp the user was last modified
+        archived:
+          type: boolean
+          description: Whether or not the user is archived
```
Comment (manas-vessel) [inline docs/pages/crm/crm.yaml:5437]: actual changes

File: docs/pages/crm/crm.yaml [hunk 2]
```diff
@@ -5405,6 +5410,7 @@ components:
           email: micheal@dundermifflin.com
           createdTime: "2019-08-24T14:15:22Z"
           modifiedTime: "2019-08-24T14:15:22Z"
+          archived: false
```
Comment (manas-vessel) [inline docs/pages/crm/crm.yaml:5413]: changes

File: docs/pages/crm/crm.yaml [hunk 3]
```diff
@@ -61,6 +61,11 @@ paths:
           in: query
           name: allFields
           description: Returns all fields including non-unifiable and custom fields under the "additional" property in the response
+        - schema:
+            type: boolean
+          in: query
+          name: archived
+          description: Set to true to include archived users in the response. Defaults to false.
```
Comment (manas-vessel) [inline docs/pages/crm/crm.yaml:68]: changes

**Review by manas-vessel — COMMENTED** [review]
Since VS code removed all the trailing spaces, I just commented where I put my changes so it's easier to review

**Review by averyyip — APPROVED** [review]

**Pull request merged**